.versionrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "packageFiles": [
+    {
+      "filename": "version.json",
+      "type": "json"
+    }
+  ],
+  "bumpFiles": [
+    {
+      "filename": "version.json",
+      "type": "json"
+    }
+  ],
+  "commitAll": true,
+  "scripts": {
+    "postbump": "npx projen && git add ."
```

Comment:
What does this do?

Reply:
After the bump script is executed then `npx projen` and `git add .` are executed. The above comment goes into more detail.

Reply:
the .versionrc.json file configures "standard-version"